aws/nextflow-compute.tf

```diff
@@ -0,0 +1,56 @@
+resource "aws_iam_instance_profile" "nf_ecs_instance_role" {
+  name = "nextflow-ecs-instance-role"
+  role = aws_iam_role.nf_ecs_role.name
+}
+
+resource "aws_batch_compute_environment" "nextflow_ondemand" {
```

Comment:
It might be nice to standardize on `nextflow_` or `nf_` so you don't have to remember when to use which.

Reply:
Yeah, I thought about this before, but with this extra prompting, I will do it.